test/DGmethods/compressible_Navier_Stokes/mms_bc_atmos.jl

```diff
@@ -198,8 +199,9 @@ let
   polynomialorder = 4
   base_num_elem = 4
 
-  expected_result = [1.5606226382564500e-01 5.3302790086802504e-03 2.2574728860707139e-04;
-                     2.5803100360042141e-02 1.1794776908545315e-03 6.1785354745749247e-05]
+  # TODO: Get solution for integration_testing = true
+  expected_result = [0.36679329773810837 0.3353991836118424 0.33691843904190333;
```

Comment:
This suggests that things are not converging.

Reply:
> #423 effects this PR

Thanks for the heads up. I'll check this out.

> This suggests that things are not converging.

That makes sense. Any idea what's wrong? Maybe this won't be converged quite as soon as I expected..

Reply:
I made a separate PR to at least remove the unnecessary fields, but we should still figure out how to get this working @jkozdon